eng/common/instructions/azsdk-tools/typespec-to-sdk.instructions.md

```diff
@@ -56,7 +56,7 @@ Your goal is to guide user through the process of generating SDKs from TypeSpec
     - Option A: "Generate SDK locally". This is currently supported only for Python. Do not recommend this for other languages.
     - Option B: "Use SDK generation pipeline"
 2. Based on selection:
-    - If Option A: Refer to #file:create-sdk-locally.instructions.md and then proceed to Step 6
+    - If Option A: Refer to #file:local-sdk-workflow.instructions.md and then proceed to Step 6
```

Comment:
should the next step be 8?

Reply:
This is a screenshot running in my vscode:
<img width="1016" height="1315" alt="image" src="https://github.com/user-attachments/assets/df385570-c734-4b50-a0ed-0d3ed67dcaba" />

Reply:
As @praveenkuttappan and @raych1 mentioned I think the longer-term goal will be for the user or agent to use this new sdk tool to generate the sdk (locally on the dev machine or on the agent machine) and then once ready submit the sdk PR. While we do use the SDK generation pipeline today we shouldn't block folks doing it themselves and we will eventually remove the SDK generation pipeline gradually (i.e. initially it will be used to generate and create the SDK PR but we will remove the PR creation from it and only use it for spec validation at some point).